langcodes/tests/test_issue_59.py

```diff
@@ -0,0 +1,21 @@
+from langcodes import closest_match
+
+
+def test_language_less_than():
+    spoken_language_1 = 'pa'
+    spoken_language_2 = 'pa-PK'
+    match = closest_match(
+        spoken_language_1, [spoken_language_2], ignore_script=True
+    )
+    print(match)
```

Comment:
I think print in tests is hard to manage on executing time, do we need the debug output? If yes, I'd prefer using [logging](https://docs.python.org/3/library/logging.html)

Reply:
Lets keep it simple for now.